scripts/convert_cogvideox_to_diffusers.py

```diff
@@ -276,11 +333,6 @@ def get_args():
         scheduler=scheduler,
     )
 
-    if args.fp16:
```

Comment:
Due to the explanation above, we shouldn't typecast all weights in the pipeline. VAE is best in FP32, text encoder could be saved in FP32 but works well at lower precisions as well, and transformer is either in BF16, or FP16 for CogVideoX-2B text-to-video

Reply:
Understood, this is the right thing to do.